controller_manager/controller_manager/controller_manager_services.py

```diff
@@ -32,7 +32,9 @@ class ServiceNotFoundError(Exception):
     pass
 
 
-def service_caller(node, service_name, service_type, request, service_timeout=0.0):
+def service_caller(
+    node, service_name, service_type, request, service_timeout=0.0, max_attempts=3
```

Comment:
Shouldn't we propagate the argument `max_attempts`? What is the exact purpose of it?

Reply:
As discussed in the WG meeting today, this is merely a method to "get the job done". The fact, that it actually might retry should not necessarily be propagated to the users' scope of parametrization.